CodingChallenges/CC_024_PerlinNoiseFlowField_Processing/CC_024_PerlinNoiseFlowField_Processing.pde

```diff
@@ -0,0 +1,55 @@
+float inc = 0.1;
```

Comment:
Would you mind adding some header comments at the top? You can look at other examples for a template. I like to link to the associated youtube url, etc.